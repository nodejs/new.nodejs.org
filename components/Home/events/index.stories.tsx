import Events from '.';
import type { Meta as MetaObj, StoryObj } from '@storybook/react';

type Story = StoryObj<typeof Events>;
type Meta = MetaObj<typeof Events>;

export const Default: Story = {};

export default { component: Events } as Meta;
