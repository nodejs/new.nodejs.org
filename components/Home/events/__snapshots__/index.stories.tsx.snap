// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Home/events Default smoke-test 1`] = `
<div class="events_container__sTJFh">
  <div class="events_image__CRtas">
    <div class="images_image__dY6kn">
      <img src="/static/images/events/openjs-world-2023/matteo.jpg"
           alt="Event illustration image"
      >
    </div>
  </div>
  <p class="events_title__ufMA1">
    OpenJS World 2023 !
    <br>
    What happened? What's next?
  </p>
  <a href="/events">
    Learn more
  </a>
</div>
`;
